eng/tools/BaselineGenerator/Program.cs

```diff
@@ -26,34 +32,57 @@ static void Main(string[] args)
 
         private readonly CommandOption _source;
         private readonly CommandOption _output;
+        private readonly CommandOption _update;
 
         public Program()
         {
-            _source = Option("-s|--source <SOURCE>", "The NuGet v2 source of the package to fetch", CommandOptionType.SingleValue);
+            _source = Option("-s|--packageSource <SOURCE>", "The NuGet source of packages to fetch", CommandOptionType.SingleValue);
```

Comment:
```suggestion
            _source = Option("-s|--package-source <SOURCE>", "The NuGet source of packages to fetch", CommandOptionType.SingleValue);
```

Nit: follow POSIX conventions for capitalization of command line args